src/app/globals.css

```diff
@@ -9,10 +9,10 @@
   --color-primary: #68fe9c;
   --color-secondary: #e2e2e2;
   --color-shade-first: #060606;
-  --color-shade-second: #1b1f24;
-  --color-shade-third: #23272c;
-  --color-shade-fourth: #393d43;
-  --color-shade-fifth: #585d64;
+  --color-shade-second: rgb(27, 31, 36, 0.4);
+  --color-shade-third: rgb(27, 31, 36, 0.4);
+  --color-shade-fourth: rgb(27, 31, 36, 0.4);
+  --color-shade-fifth: rgb(27, 31, 36, 0.4);
```

Comment:
Is it intended that these colors have an alpha channel? The background-image will be slightly visible.

Reply:
Some components have a slightly opacity in the layout. I woud like to updatet the css-globals colors later, when everyone have finished the components.